opennms-container/sentinel/.gitignore

```diff
@@ -1 +1,2 @@
+sentinel-config-schema.yml
```

Comment:
I think we can drop this one as well, -1 line 😁

Reply:
done